Backpack-SwiftUI/ImageGalleryGrid/Classes/ImageGalleryGridCategoriesCarousel.swift

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+struct ImageGalleryGridCategoriesCarousel<ImageView: View>: View {
+    let categories: [BPKImageGalleryCategory<ImageView>]
+    @Binding var selectedCategory: Int
+    private let categoryImageSideLength: CGFloat = 90
+    
+    var body: some View {
+        ScrollView(.horizontal) {
+            HStack(alignment: .top) {
+                ForEach(0..<categories.count, id: \.self) { ndx in
+                    categoryThumb(for: categories[ndx])
```

Comment:
could you wrap the categoryThumb in a Button and remove the onTapGesture? the other approach would be to add an accessibitiy trait 'isButton' so that VoiceOver reads this as a button

Reply:
I'm not sure what you mea by "wrap the categoryThumb in a Button". I've tried:
```
ZStack {
                        categoryThumb(for: categories[ndx])
                        
                        BPKButton("") {
                            selectedCategory = ndx
                        }.frame(
                            width: categoryImageSideLength,
                            height: categoryImageSideLength
                        ).fixedSize()
                    }
```
And the results are not good!

Reply:
I meant more like this: but you went with the onTap approach so it's fine as is
```swift
Button(action: {}) {
  categoryThumb()
}